daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -416,6 +416,8 @@ app.post('/dsstatus', (req, res) => {
 
 {{< /tabs >}}
 
+Some pubsub components (e.g.: Redis) will redeliver a message if a response is not sent back within a specified time window. For redis, the default is 60 sec and can be configured through the processingTimeout setting in component's meta or set to 0 to disable this behavior.
```

Comment:
Looks like this note belongs in the redis component reference rather in this how-to since it is specific to that implementation. I don't see why this should be added here.

Reply:
Hmm it might be nice to have a generic note though, to tell users that this behavior may happen. Maybe something like:
```suggestion
{{% alert title="👷‍♂️ Under construction 🚧" color="primary" %}}
Some pubsub components (e.g. Redis) will redeliver a message if a response is not sent back within a specified time window. Make sure to configure metadata such as `processingTimeout` to customize this behavior. For more information refer to the respective [component references]({{< ref supported-pubsub >}}).
{{% /alert %}}
```